File: Sesion8_androidStudio/app/src/main/java/com/example/sesion8tarea/MainActivity.java
package com.example.sesion8tarea;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button fra;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        fra= findViewById(R.id.idfragmentos);
    }

    public void fragementado(View view){
        Intent fragmen = new Intent(this, fragmentos.class);
        startActivity(fragmen);
    }

    //botn de la lista
    public void ListView(View view){
        Intent camerun = new Intent(this, lista.class);
        startActivity(camerun);
    }

}

File: sesion9-androidStudio/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun goToSecondActivity(view: View) {
        val intent = Intent(this, LinearLayout::class.java)
        startActivity(intent)
    }

    fun relative(view: View) {
        val intent = Intent(this, Relative::class.java)
        startActivity(intent)
    }
    fun contraint(view: View) {
        val intent = Intent(this, ConstraintLayout::class.java)
        startActivity(intent)
    }
    fun table(view: View) {
        val intent = Intent(this, Tablelayout::class.java)
        startActivity(intent)
    }


}
